README.md

```diff
@@ -94,16 +95,21 @@ All the actions return two values: result and error. Results can be either`balle
 ##### Example
 * Request 
 ```ballerina
-    jira:JiraConnector jiraConnector = {};
+    
     jira:JiraConnectorError jiraError = {};
     jira:Project project = {};
     string projectKey = "RRDEVSPRT";
     
+    //Creating the jira Connector as an endpoint
+    endpoint jira:JiraConnectorEndpoint jiraConnectorEP {
+        base_url:"https://support-staging.wso2.com"
+    };
+    
     //Authentication 
-    var response = jiraConnector.authenticate("ashan@wso2.com", "ashan123");
+    var response = jiraConnector -> authenticate("ashan@wso2.com", "ashan123");
```

Comment:
Move authentication to initializer. Why? If I want to login to connector in same runtime using two users they will over ride each other's authentication details.

Reply:
Need to update authetnication.